uportal-war/src/main/java/org/jasig/portal/io/PortletPreferenceOverridePhrase.java

```diff
@@ -1,62 +0,0 @@
-/**
- * Licensed to Apereo under one or more contributor license
- * agreements. See the NOTICE file distributed with this work
- * for additional information regarding copyright ownership.
- * Apereo licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file
- * except in compliance with the License.  You may obtain a
- * copy of the License at the following location:
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.jasig.portal.io;
-
-import org.danann.cernunnos.EntityConfig;
-import org.danann.cernunnos.Formula;
-import org.danann.cernunnos.Phrase;
-import org.danann.cernunnos.Reagent;
-import org.danann.cernunnos.ReagentType;
-import org.danann.cernunnos.SimpleFormula;
-import org.danann.cernunnos.SimpleReagent;
-import org.danann.cernunnos.TaskRequest;
-import org.danann.cernunnos.TaskResponse;
-
-public class PortletPreferenceOverridePhrase implements Phrase {

```

Comment:
@apetro -- Sort of.  I'm glad you asked, because I want this part to be broadly known & abundantly clear...

Before ANYTHING was removed, multiple layers of diligence were applied to verify that the item is genuinely unused.  The *first* of these layers was almost always a '$grep -R "{item}" uportal-war/src/main'.  Even that was often followed by a '$mvn clean' plus '$grep -R "{item}" .'.  Any Cernunnos Phrase that is in use would have generated a hit in uportal-war/src/main.

In addition to that, the following diligence steps were applied to everything in the PR (sometimes in batches):
* $ant clean initportal // NB:  tests are run
* rm -rf export/
* $ant data-export -Ddir=export // Export every object in the portal
* $ls export // Verify that every entity type seems to be exporting as usual
* // Start the portal, log in as admin, faculty, staff, and student
* // Verify the portal log generates no additional WARNings

I used other tactics beyond these.  Anything with Spring annotations was treated with the utmost suspicion (I'm not sure there was any of that removed, in the end).  Classes that implement interfaces (or abstract classes) were treated with additional suspicion.

So in a way, yes:  if these things had not been done, you should have been deeply worried.  This PR would not have passed the CI checks if these steps had not been taken.